Readme.md

```diff
@@ -3,22 +3,39 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E499)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to show and hide a popup window via server side code
+# Popup Control for ASP.NET Web Forms - How to show and hide a popup window at the server side
```

Comment:
- Why do you use **at**?
- **popup** is an adjective: pop-up

Reply:
Can we write the word "pop-up" in the phrase "Popup Control" without a hyphen and with a capital letter (because it's the name of the ASPxPopupControl),  but with a hyphen in the phrase "pup-up window" ?

```suggestion
# Popup Control for ASP.NET Web Forms - How to show and hide a pop-up window on the server side
```

Reply:
Yes